src/basics/CopyArray.java

```diff
@@ -0,0 +1,16 @@
+package basics;
+
+import java.util.Arrays;
+
+public class CopyArray {
+  public static void main(String[] args) {
+    String[] names = {"Anna", "Hanna", "Sanna", "Jenna"};
+    String[] copy = new String[4];
```

Comment:
Use `names.length` instead of `4`, this makes your code more robust, as it'll still work in case the number of elements in the original array changes.